tests/test_surface_events.cpp

```diff
@@ -588,4 +588,70 @@ TEST_F(ClientSurfaceEventsTest, buffer_release)
     EXPECT_TRUE(buffer_released[2]);
 }
 
+TEST_F(ClientSurfaceEventsTest, surface_gets_enter_event)
+{
+    using namespace testing;
+
+    Test::RecordProperty("wlcs-skip-test", "Mir does not send enter events yet");
```

Comment:
We should not be implementing Mir specific limitations in these tests.

Any tests that Mir expects to fail ought to be "xfail"ed by the Mir integration code.

Reply:
fixed